73063.typ

```diff
@@ -30,7 +30,7 @@
 		([Brakes], [TEST and SET]),
 		([Seats, belts, harnesses], [SECURE]),
 		([Fuel valve], [BOTH]),
-		([Radios, electrical equipment], [OFF]),
+		([Radios, autopilot, electric devices], [OFF]),
```

Comment:
"electrical equipment" was changed to "electric devices" to make this fit on one line (here and in the Shutdown checklist)